index.bs

```diff
@@ -268,6 +276,8 @@ spec:url; type:dfn; text:scheme
 
   The <dfn attribute for="HTMLPortalElement">src</dfn> IDL attribute must [=reflect=] the <{portal/src}> content attribute.
 
+  The <dfn attribute for="HTMLPortalElement">referrerPolicy</dfn> IDL attribute must [=reflect=] the <{portal/referrerpolicy}> content attribute, [=limited to only known values=].
```

Comment:
Should we specify the empty string defaulting to noreferrer for portals? I think it makes more sense to make the referrer explict, and the spec https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-empty-string says that the empty string falls back to a referrer policy defined elsewhere.

Reply:
The precedent everywhere else is that the source browsing context (the one that requested the fetch/subresource)'s referrer policy is used in such cases. It's written in a way that is rather implicit.

That does mean that this is a behavior change, since if the outer document's policy is not no-referrer and the attribute is omitted, we will start sending the referrer.

Reply:
While that would be a behavior change, I think it's a positive change form a privacy standpoint. By making the referrer policy more explicit, it gives us privacy by default unless otherwise specified. Given that we don't need to support legacy applications, we should take the opportunity to improve this behavior. As another precedent, we also only support use noopener.

If we are to change that due to ongoing API feedback, it would be much easier to change from disallowed by default than otherwise.

This was also raised in issue #123.

Reply:
I'm kinda on the fence about it. We *definitely* should make it easy to send no referrer, in any case. If we want to allow sending a referrer (especially in the 1P case it doesn't seem unreasonable to want it), putting the policy in the hands of the author is the consistent way to do that.

It might be surprising if `<portal>` were the unique element that doesn't respect ambient policy (but only the policy on the element itself, defaulting to `no-referrer`); I'm kinda inclined to defer to @domenic's expertise on the relative costs of a more restrictive default vs a more consistent default.

Reply:
`noopener` is slightly different as we have other technical reasons to want to avoid exposing the opener `Window`. :)

Reply:
SGTM. I'll defer to @domenic's expertise here on the best behavior to standardize/engage with other browsers.